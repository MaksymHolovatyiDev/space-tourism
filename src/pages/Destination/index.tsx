import {PlanetPanel} from 'widgets/planet-panel';
import styles from './styled.module.scss';

function Destination() {
  return (
      <main className={styles.root}>
        <PlanetPanel />
      </main>
  );
}

export default Destination;
