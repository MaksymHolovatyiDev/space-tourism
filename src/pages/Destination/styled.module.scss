.root {
  height: 100vh;
  width: 100vw;

  display: flex;
  justify-content: space-between;
  align-items: flex-end;

  padding: 0 163px 112px 230px;

  background-size: cover;
  background-repeat: no-repeat;
  background-image: url('./assets/background-destination-desktop.jpg');

  @media screen and (max-width: 1300px) {
    flex-direction: column;

    justify-content: center;
    align-items: center;

    gap: 53px;

    padding: 220px 98px 62px 97px;

    background-image: url('./assets/background-destination-desktop.jpg');
  }

  @media screen and (max-width: 1300px) {
    flex-direction: column;

    justify-content: center;
    align-items: center;

    gap: 26px;

    padding: 139px 24px 58px;

    background-image: url('./assets/background-destination-mobile.jpg');
  }
}
