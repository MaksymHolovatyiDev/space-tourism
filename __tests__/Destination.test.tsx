import renderer from 'react-test-renderer';
import Destination from '../src/pages/Destination';
import React from 'react';

it('renders correctly', () => {
  const tree = renderer.create(<Destination />).toJSON();
  expect(tree).toMatchSnapshot();
});
