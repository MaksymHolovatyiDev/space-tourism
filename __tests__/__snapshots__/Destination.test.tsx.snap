// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<main>
  <img
    alt="Moon"
    src="src/entities/planet/assets/image-moon.webp"
  />
  <div>
    <ul>
      <li
        onClick={[Function]}
      >
        <div
          className="undefined undefined undefined"
        >
          <p
            className="undefined undefined"
          >
            Moon
          </p>
        </div>
      </li>
      <li
        onClick={[Function]}
      >
        <div
          className="undefined undefined false"
        >
          <p
            className="undefined undefined"
          >
            Mars
          </p>
        </div>
      </li>
      <li
        onClick={[Function]}
      >
        <div
          className="undefined undefined false"
        >
          <p
            className="undefined undefined"
          >
            EUROPA
          </p>
        </div>
      </li>
      <li
        onClick={[Function]}
      >
        <div
          className="undefined undefined false"
        >
          <p
            className="undefined undefined"
          >
            TITAN
          </p>
        </div>
      </li>
    </ul>
    <div
      className="undefined"
    >
      <div>
        <h2
          className="undefined undefined undefined"
        >
          Moon
        </h2>
        <p
          className="undefined undefined"
        >
          See our planet as you’ve never seen it before. A perfect relaxing trip away to help regain perspective and come back refreshed. While you’re there, take in some history by visiting the Luna 2 and Apollo 11 landing sites.
        </p>
      </div>
      <div>
        <div>
          <p
            className="undefined undefined undefined"
          >
            avg. distance
          </p>
          <p
            className="undefined undefined undefined"
          >
            384,400 km
          </p>
        </div>
        <div>
          <p
            className="undefined undefined undefined"
          >
            est. travel time
          </p>
          <p
            className="undefined undefined undefined"
          >
            3 days
          </p>
        </div>
      </div>
    </div>
  </div>
</main>
`;
